src/components/input/input.jsx

```diff
@@ -50,6 +50,9 @@ const Input = forwardRef(function Input(props, forwardedRef) {
     },
   }));
 
+  const describedBy = [`${props.id}Hint`];
```

Comment:
A recent pattern we've been kinda liking for managing IDs, refs, etc.

Use a single object variable to keep track of all IDs with respect to the layout/roles. e.g.

```js
const ids = {
  container: props.id,
  label: `${props.id}-label`,
  tooltip: `${props.id}-tooltip`,
  validationMessage: `${props.id}Hint`,
};
```

Up to you if you want to also repeat the pattern.